sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -104,7 +104,7 @@ AdaptiveSparkPlan (8)
 (1) Scan parquet spark_catalog.default.explain_temp1
 Output [2]: [key#x, val#x]
 Batched: true
-Location [not included in comparison]/{warehouse_dir}/explain_temp1]
+Location: InMemoryFileIndex [file:[not included in comparison]/{warehouse_dir}/explain_temp1]
```

Comment:
It turns out that we don't need the `Location` regex at all, as it hides too much information.